### DISCUSSION TITLE
Estructura menú sitio de la comunidad

### DISCUSSION THREAD

**Discussion by ecabrerar** [JavaDominicano/java-dominicano-comite-ejecutivo#21] [Ideas]
Para el sitio de la comunidad, entiendo que inicialmente podemos tener algo como esto:

Home Page
Banner con descripción sobre la comunidad.

Menu
- Sobre la comunidad
- Contactos
- Eventos
- Comunidad
   - Conoce el equipo  
    -Get Help (Asistencia Técnica / Ayuda Técnica)
    -Donate (Become a Sponsor / Get Involved / Make a Donation / Sponsors - Donors )
    
Abro debate sobre cuales opciones podemos tener en el menu principal

**Comment by fredpena** [comment]
### Esto es solo una idea, podemos coger lo que nos convenga, y dejar lo demás, o agendarlo para planes futuros. 

**Inicio:**
_- Breve introducción a la comunidad y su misión._
_- Actualizaciones recientes y noticias importantes._
_- Enlaces a recursos clave para los miembros._

**Eventos:**
_- Próximos eventos: Lista de eventos planificados con fechas, lugares y detalles._
_- Eventos pasados: Resúmenes, fotos y enlaces a las grabaciones de eventos anteriores._
_- Calendario: Calendario interactivo con todas las fechas de los eventos._

**Speakers:**
_- Lista de speakers: Perfiles y biografías de los speakers que han participado en eventos._
_- Detalles de presentaciones: Información sobre las charlas y talleres que han dado._

**Recursos:**
_- Tutoriales: Enlaces a tutoriales y guías relacionados con la tecnología Java._
_- Artículos: Publicaciones sobre tendencias, novedades y mejores prácticas en Java._
_- Libros recomendados: Lista de libros recomendados para desarrolladores de Java._
_- Herramientas y frameworks: Recomendaciones de herramientas y frameworks populares._

**Comunidad:**
_- Foro de discusión: Espacio para que los miembros puedan interactuar y hacer preguntas._
_- Colaboraciones: Oportunidades para contribuir a proyectos y colaborar con otros miembros._
_- Redes sociales: Enlaces a perfiles de redes sociales y grupos en línea._

**Acerca de Nosotros:**
_- Información sobre la comunidad: Historia, valores y objetivos._
_- Equipo: Perfiles de los organizadores y líderes de la comunidad._

**Contacto:**
_- Formulario de contacto: Para consultas y retroalimentación._
_- Información de contacto: Dirección de correo electrónico y redes sociales._

**Únete:**
_- Información sobre cómo unirse a la comunidad._
_- Instrucciones para registrarse en eventos y participar en actividades._

**Patrocinadores:**
_- Reconocimiento a los patrocinadores y colaboradores de la comunidad._

**Galería:**
_- Fotos y videos de eventos pasados y momentos destacados._

**Blog:**
_- Publicaciones regulares sobre novedades en tecnología Java, tutoriales, opiniones, etc._

**Comment by vacax** [comment]
Estoy de acuerdo con lo planteado por @fredpena.
